src/pages/CommitDetailPage/subRoute/FilesChangedTab/FilesChangedTable/FilesChangedTable.jsx

```diff
@@ -157,6 +157,8 @@ function FilesChangedTable() {
   })
 
   const flags = queryParams?.flags?.length > 0 ? queryParams?.flags : undefined
+  const components =
```

Comment:
Do we need changes in this file? `useCommit` doesn't make use of components.

Reply:
Added to `UseCommitArgs`